src/App.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from 'react';
+import { UserList } from './components/List/List';
+
+function App() {
+    const [users, setUsers] = useState([]);
+
+    const getData = async () => {
+        const response = await fetch('https://jsonplaceholder.typicode.com/users');
+        const data = await response.json();
+        setUsers(data);
+    };
+
+    return (
+        <div className="App">
+            <UserList onClick={getData}></UserList>
+            {!users.length ? (
+                <div>Todavia no tenemos usuarios</div>
+            ) : (
+                <>
+                    {users.map((user) => (
+                        <div key={user.username}>
+                            <img
+                                alt={user.username + '.logo.svg'}
+                                src={`https://avatars.dicebear.com/api/male/${user.username}.svg`}
+                                style={{ width: 50, height: 50 }}
+                            />
+                            <div>{user.name}</div>
+                            <div>{console.log(user.username)}</div>
```

Comment:
@nahomic could you explain me why do you have a console.log in the render() ? ??